.travis.yml

```diff
@@ -12,8 +12,9 @@ cache:
 jobs:
   include:
 
+    - stage: "Supported Platforms and Compilers"
```

Comment:
Maybe this should be "Compile and Test", otherwise it is going to look really weird when is it displayed along "Release builds".